README.md

```diff
@@ -1 +1,83 @@
-# CytoPipe
\ No newline at end of file
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [CytoPipe](#cytopipe)
+- [Formatting](#formatting)
+  - [smkfmt](#smkfmt)
+- [Testing](#testing)
+  - [Dry Runs](#dry-runs)
+
+# CytoPipe
+
+# Formatting
+
+## smkfmt
```

Comment:
This says `smkfmt` but below you write `snakefmt` - just checking if this is correct!

Reply:
Typo fixed! : https://github.com/WayScience/CytoPipe/pull/5/commits/7564f7a4b657c4dd35ce6e0d8705d305845f0926